gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -49,15 +52,17 @@
  * <title>Example launch line with complex if condition</title>
  * gst-launch ... (some tensor stream)
  *      ! tensor_filter framework=custom name=your_code.so
- *      ! tensor_if compared_value=A_VALUE
- *          compared_value_option=0:0:0:0,0 # 1st tensor's [0][0][0][0].
- *          operator=EQ
- *          supplied_values=1
- *          then=PASSTHROUGH # or whatsoever you want
- *          else=SKIP # or whatsoever you want
- *      ! tensor_demux name=d
- *        d.src_0 ! queue ! fakesink # throw away the 1/0 value.
- *        d.src_1 ! queue ! do whatever you want here...
+ *      ! tensor_if name=tif \
+ *          compared_value=A_VALUE \
+ *          compared_value_option=0:0:0:0,0 \# 1st tensor's [0][0][0][0].
+ *          operator=EQ \
+ *          supplied_values=1 \
+ *          then=PASSTHROUGH \# or whatsoever you want
+ *          else=SKIP \# or whatsoever you want
+ *      ! tif.src_0 ! tensor_demux name=d \ #for TRUE action
+ *         d.src_0 ! queue ! fakesink # throw away the 1/0 value.
+ *         d.src_1 ! queue ! do whatever you want here...
+ *      ! tif.src_1 ! (tensor(s) stream for FALSE action) ...
```

Comment:
If it is PASSTHROUGH and SKIP, how can we have src_1?

Reply:
Right. I will remove it and clear example script.